src/service/memberHandler/postMember.jsx

```diff
@@ -3,27 +3,27 @@ import { appSettings } from "../../Appdata/appdata";
 import { toast } from "react-toastify";
 import deleteImageHandler from "../deleteImageService";
 
-export default function submitMember(memberData, profileURL) {
-  return axios
-    .post(`${appSettings.member}s`, memberData)
-    .then((response) => {
-      if (response.status === 201) {
-        // Member Added Successfully
-        toast.success("Member Added Successfully");
-
-        console.log(response.data); // log the response message to the console
-        return response.data;
-      }
-    })
-    .catch(async (error) => {
-      await deleteImageHandler(profileURL);
-      if (error.response && error.response.status === 406) {
-        toast.error("Member Name already exists");
-      } else if (error.response && error.response.status === 500) {
-        toast.error("Internal Server Error");
-      } else {
-        toast.error("Failed to add member");
-      }
-      throw error;
-    });
+export default async function submitMember(memberData) {
+  try {
+    const response = await axios.post(`${appSettings.member}s`, memberData);
```

Comment:
I think same ra ni sa taas it's also calling an axios API call. We can refactor and store it sa API folder under membersAPI then save it as postMemberAPI().